### PR TITLE
DSE 6.8.25

### PR DESCRIPTION
**Pull request by tiagomlalves**

### CODE REVIEW

File: DSE_6.8_Release_Notes.md [hunk 1]
```diff
@@ -2,6 +2,52 @@
...
+* Fixed the cqlsh tool to restore the ability to use the EXECUTE AS command. (DSP-22417)
+* Fixed bug with overriding customer limits.d file if there are new values in the package upgrade. Complementary to DSP-21928. (DSP-22594)
+
+## 6.8.25 DSE Cassandra
+* Changed outdated `nofile` parameter value to `1048576` in `/etc/security/limits.d/cassandra.conf`. (DSP-21947)
+
+## 6.8.25 DSE Search
+* Fixed parsing range queries on DSE Search indexed columns of CQL types Date and Time. (DSP-22548)
+* Added index size metric to Solr Core. (DSP-22546)
+* Added `numDocs`, `maxDoc`, and `deletedDocs` metrics to Solr Core. (DSP-22587)
```
Comment (tlasica) [inline DSE_6.8_Release_Notes.md:30]: same here

File: DSE_6.8_Release_Notes.md [hunk 2]
```diff
@@ -2,6 +2,52 @@
...
+* Removed the PMD plugin from gradle config. (DSP-22575, [CVE-2019-7722](https://nvd.nist.gov/vuln/detail/CVE-2019-7722))
+* Fixed the cqlsh tool to restore the ability to use the EXECUTE AS command. (DSP-22417)
+* Fixed bug with overriding customer limits.d file if there are new values in the package upgrade. Complementary to DSP-21928. (DSP-22594)
+
+## 6.8.25 DSE Cassandra
+* Changed outdated `nofile` parameter value to `1048576` in `/etc/security/limits.d/cassandra.conf`. (DSP-21947)
+
+## 6.8.25 DSE Search
+* Fixed parsing range queries on DSE Search indexed columns of CQL types Date and Time. (DSP-22548)
+* Added index size metric to Solr Core. (DSP-22546)
```
[the remark this answers is not in the document]
Comment (tlasica) [reply DSE_6.8_Release_Notes.md:29]: ok, lets leave it as is.

File: DSE_6.8_Release_Notes.md (hunk 1 above)
Comment (tlasica) [reply DSE_6.8_Release_Notes.md:30]: lets leave it as is and see.

**Review by tlasica — APPROVED** [review]

File: DSE_6.8_Release_Notes.md [hunk 3]
```diff
@@ -2,6 +2,52 @@
...
+## 6.8.25 DSE Graph
+* Fixed an error when using the search index and traversing with an inequality operator such as gt or gte against Date or Time data types. (DSP-21279)
+
+## 6.8.25 DSE Management API
+* Improved Management API logic to reduce the likelihood of resource leaks when DSE is starting up. (DSP-22539)
+
+## 6.8.25 DSE Security
+* Fixed `EXECUTE AS` functionality to work with RLAC. (DSP-22508)
+
+## 6.8.25 DSE CVE
```
[the remark this answers is not in the document]
Comment (tiagomlalves) [reply DSE_6.8_Release_Notes.md:47]: sure!

[the remark this answers is not in the document]
Comment (tiagomlalves) [reply DSE_6.8_Release_Notes.md:47]: If we move this ticket to `Security` shoudn't it have CVE ID? We upgraded it to solve CVE that were reported, didn't we?

[the remark this answers is not in the document]
Comment (tlasica) [reply DSE_6.8_Release_Notes.md:47]: ah, as usual - this is not so simple. it removes CVE from this source, but another source of same CVE remains => CVE is still present in the DSE.

[the remark this answers is not in the document]
Comment (tiagomlalves) [reply DSE_6.8_Release_Notes.md:47]: I propose to just leave it where it is then

**Pull request merged**